src/applications/mhv/landing-page/app-entry.jsx

```diff
@@ -1,14 +1,15 @@
 import 'platform/polyfills';
 import './sass/mhv-landing-page.scss';
 
-import startApp from 'platform/startup';
+import startApp from 'platform/startup/router';
```

Comment:
```suggestion
import { startAppFromIndex as startApp } from '@department-of-veterans-affairs/platform-startup/exports';```

Reply:
I ended up using `import { startAppFromRouter as startApp } from '@department-of-veterans-affairs/platform-startup/exports'` here.